### PR TITLE
Sort

### PR DESCRIPTION
**Pull request by Brettj85**

### CODE REVIEW

File: sort/Program.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                    if (arr[j] > arr[min])
+                    {
+                        min = j;
+                        Swap(ref arr[i], ref arr[min]);
+                    }
+                }
+            }
+        }
+
+        private static void Swap(ref int x, ref int y)
```
Comment (daniefer) [inline sort/Program.cs]: Nice and good use of `ref`

File: sort/Program.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            PrintResults(arr);
+        }
+
+        private static void Sort(int[] arr)
+        {
+            int i, j, min;
+            for (i = 0; i < arr.Length; i++)
+            {
+                min = i;
+                for (j = 0; j < arr.Length; j++)
```
Comment (daniefer) [inline sort/Program.cs]: You should try setting `j = i` instead of this `min = i` stuff...

File: multithreading/Program.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AsyncSort
```
Comment (daniefer) [inline multithreading/Program.cs]: Good try. We will get there. Got to understand the basics before we start in on async and parallel stuff.

Comment (Brettj85) [reply multithreading/Program.cs]: yea im still a newb ha ha but i learned A LOT while trying to do this.. so least i got a little out of it.

**Pull request merged**